src/library_webgpu.js

```diff
@@ -1491,7 +1491,7 @@ var LibraryWebGPU = {
   wgpuQueueOnSubmittedWorkDone: function(queueId, signalValue, callback, userdata) {
     var queue = WebGPU.mgrQueue.get(queueId);
 #if ASSERTIONS
-    assert(signalValue_low === 0 && signalValue_high === 0, 'signalValue not supported, must be 0');
+    assert(signalValue === 0, 'signalValue not supported, must be 0');
```

Comment:
If `signalValue` can be a BigInt then this should be `==`
```suggestion
    assert(signalValue == 0, 'signalValue not supported, must be 0');
```
(I'm sure this won't be the only bug, but I spotted it now, so may as well fix it 🙂)